### PR TITLE
[IMP] codeowners: add codeowners for logistics team

### PR DESCRIPTION
**Pull request by ryv-odoo**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -110,3 +110,36 @@
...
+/addons/purchase_requisition/ @odoo/rd-logistics
+/addons/purchase_requisition_stock/ @odoo/rd-logistics
+
+/addons/sale_mrp/ @odoo/rd-logistics
+/addons/sale_stock/ @odoo/rd-logistics
+/addons/sale_stock_margin/ @odoo/rd-logistics
+/addons/sale_purchase/ @odoo/rd-logistics
+/addons/sale_purchase_stock/ @odoo/rd-logistics
+
+/addons/repair/ @odoo/rd-logistics
```
Comment (seb-odoo) [inline .github/CODEOWNERS:145]: Hello,

Thank you for the update, explicit list looking good.

1. Don't add modules that don't exist in v12 here. Handle new modules progressively as forward ports go (and include the new one in master only when it is merged).
2. Please read and follow the instructions in the file, in particular those should be ordered alphabetically with the other modules, not added in a logistics category.

Comment (ryv-odoo) [reply .github/CODEOWNERS]: Thanks, I updated my commit to:
- Respect the alphabetically order
- Remove the unexciting module (I will be add gradually with forward ports)

**Review by seb-odoo — APPROVED** [review]
The diff looks ok to me now.
I didn't test the existence of each module specifically, I'm trusting you did it correctly.

@robodoo delegate+

Reminder you will have to properly set up the @odoo/rd-logistics team if it is not done already. In particular it needs to have write access on the repository (both the team itself, and each member that intend to be added as codeowner receiver), needs to be done by a github admin of Odoo. Also there are specific codeowners config on the team settings itself that you can change.

**Comment by ryv-odoo** [comment]
@robodoo r+

**Pull request closed without merging**